doc/source/text.rst

```diff
@@ -306,20 +306,20 @@ The same alignment can be used when ``others`` is a ``DataFrame``:
 Concatenating a Series and many objects into a Series
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All one-dimensional list-likes can be arbitrarily combined in a list-like container (including iterators, ``dict``-views, etc.):
+An arbitrary combination of ``Series``, ``Index`` and ``np.ndarray`` (1-dimensional) can be passed in a list-like container (including iterators, ``dict``-views, etc.):
 
 .. ipython:: python
 
     s
     u
-    s.str.cat([u.values, ['A', 'B', 'C', 'D'], map(str, u.index)], na_rep='-')
+    s.str.cat([u.values, u, u.index.astype(str)], na_rep='-')
 
 All elements must match in length to the calling ``Series`` (or ``Index``), except those having an index if ``join`` is not None:
 
 .. ipython:: python
 
     v
-    s.str.cat([u, v, ['A', 'B', 'C', 'D']], join='outer', na_rep='-')
+    s.str.cat([u, v, u.values], join='outer', na_rep='-')
```

Comment:
same here

Reply:
In this case, `u` and `u.values` will have been shown 3 lines above, and `v` directly above. I think this should be plenty, no?

Reply:
use w as you defined above

Reply:
The point here is to show how `join` works with objects that have no index, so can't use `w` (which has an index).